chapter-06/start/States/GameplayState.cs

```diff
@@ -26,6 +37,33 @@ public override void HandleInput()
             {
                 NotifyEvent(Events.GAME_QUIT);
             }
+
+            if (state.IsKeyDown(Keys.Left))
+            {
+                _playerSprite.MoveLeft();
+                KeepPlayerInBounds();
+            }
+
+            if (state.IsKeyDown(Keys.Right))
+            {
+                _playerSprite.MoveRight();
+                KeepPlayerInBounds();
+            }
+        }
+
+        private void KeepPlayerInBounds()
+        {
+            if (_playerSprite.Position.X < 0)
```

Comment:
Brace use for the conditionals (even single line) - I believe I had established that earlier

Reply:
I have noticed your braced single line bodies. For some reason it didn't compute in my head that I should follow that style. Fixing.

Reply:
fixed